src/test_typing_extensions.py

```diff
@@ -7165,13 +7165,58 @@ def test_attributes(self):
         self.assertEqual(ListOrSetT.__type_params__, (T,))
         self.assertEqual(ListOrSetT.__parameters__, (T,))
 
+        subscripted = ListOrSetT[int]
+        self.assertEqual(subscripted.__name__, "ListOrSetT")
+        self.assertEqual(subscripted.__value__, Union[List[T], Set[T]],)
+        self.assertEqual(subscripted.__type_params__, (T, ))
+        self.assertEqual(subscripted.__parameters__, ())
+
+        T2 = TypeVar("T2")
+        subscriptedT = ListOrSetT[T2]
+        self.assertEqual(subscriptedT.__name__, "ListOrSetT")
+        self.assertEqual(subscriptedT.__value__, Union[List[T], Set[T]],)
+        self.assertEqual(subscriptedT.__type_params__, (T, ))
+        self.assertEqual(subscriptedT.__parameters__, (T2, ))
+
         Ts = TypeVarTuple("Ts")
         Variadic = TypeAliasType("Variadic", Tuple[int, Unpack[Ts]], type_params=(Ts,))
         self.assertEqual(Variadic.__name__, "Variadic")
         self.assertEqual(Variadic.__value__, Tuple[int, Unpack[Ts]])
         self.assertEqual(Variadic.__type_params__, (Ts,))
         self.assertEqual(Variadic.__parameters__, tuple(iter(Ts)))
 
+        # Test bare
+        subscripted_tuple = Variadic[int, float]
+        self.assertEqual(subscripted_tuple.__name__, "Variadic")
+        self.assertEqual(subscripted_tuple.__value__, Tuple[int, Unpack[Ts]])
+        self.assertEqual(subscripted_tuple.__type_params__, (Ts,))
+        self.assertEqual(subscripted_tuple.__parameters__, ())
+
+        # Test with Unpack
+        subscripted_tupleT = Variadic[Unpack[Tuple[int, T]]]
+        self.assertEqual(subscripted_tupleT.__name__, "Variadic")
+        self.assertEqual(subscripted_tupleT.__parameters__, (T, ))
+
+        # Test with Unpack and TypeVarTuple
+        subscripted_Ts = Variadic[Unpack[Ts]]
+        self.assertEqual(subscripted_Ts.__parameters__, (Ts, ))
+
+        # Use with Callable
+        # Use with Callable+Concatenate
+        subscripted_callable_concat = Variadic[Callable[Concatenate[Literal["s"],  P], T]]
+        self.assertEqual(subscripted_callable_concat.__parameters__, (P, T))
+
+        subcriped_callable_tvt = Variadic[Callable[[Unpack[Ts]], T]]
+        self.assertEqual(subcriped_callable_tvt.__parameters__, (Ts, T))
+
+        # Use with Callable+Unpack
+        CallableTs = TypeAliasType("CallableTs", Callable[[Unpack[Ts]], Any], type_params=(Ts, ))
+        self.assertEqual(CallableTs.__type_params__, (Ts,))
+        self.assertEqual(CallableTs.__parameters__, (*Ts,))
+
+        unpack_callable = CallableTs[Unpack[Tuple[int, T]]]
+        self.assertEqual(unpack_callable.__parameters__, (T,))
+
```

Comment:
After restructuring I thought why not make a comprehensive test to test alias-arg combinations.
It is compact and kind of complete, on the other hand it increases the total amount of tests and skipped tests. I am not sure if this is wanted.